app/models/krikri/provider.rb

```diff
@@ -1,80 +1,173 @@
 module Krikri
   ##
-  # This models provider data from the {Krikri::Repository}.
+  # Provider is an ActiveModel object. It represents a provider that has been
+  # indexed in Solr.
   #
-  # @todo create an appropriate `ActiveTriples::PersistenceStrategy' for
-  #   handling providers as data from a repository alongside LDP data.
-  # @see ActiveTriples::Resource
-  class Provider < DPLA::MAP::Agent
-    configure :base_uri => Krikri::Settings.prov.provider_base
+  # This is a read-only object. It does not write new providers to Solr, nor
+  # does it update or delete providers.
+  #
+  # A DPLA::MAP::Agent object can be constructed from an instance of this
+  # ActiveModel object using the :agent method. The DPLA::MAP::Agent object
+  # interacts with Marmotta, rather than Solr. If the :rdf_subject is known,
+  # create the ActiveModel instance with #new (instead of #find) to avoid an
+  # unnecessary call to Solr.
+  #   @example Krikri::Provider.new(rdf_subject: 'http://blah.com/123').agent
+  #     => [DPLA::MAP::Agent] 
+  #
+  class Provider
+    include ActiveModel::Validations
+    include ActiveModel::Conversion
+    extend ActiveModel::Naming
+
+    ##
+    # @attr :rdf_subject [String] the URI identifying the provider
```

Comment:
We might consider using the [`@!attribute` tag](http://www.rubydoc.info/gems/yard/file/docs/Tags.md#attribute). `@attr` is deprecated.